src/components/Header/Header.jsx

```diff
@@ -14,16 +14,12 @@ export default function Header() {
   }
   return (
     <S.Header>
-      <S.LeftWrapper>
-        <ScoreDisplay />
-      </S.LeftWrapper>
-      <S.CenterWarpper>
-        <Logo />
-      </S.CenterWarpper>
+      <ScoreDisplay />
+      <Logo />
+     <menu />
```

Comment:
menu needs to be in captial M. Its a component

Reply:
and this should be your menu component

Reply:
done